networkdb/broadcast.go

```diff
@@ -168,7 +167,6 @@ func (nDB *NetworkDB) sendTableEvent(event TableEvent_Type, nid string, tname st
 		id:    nid,
 		tname: tname,
 		key:   key,
-		node:  nDB.config.NodeID,
```

Comment:
Just to confirm, the node attribute is just redundant, I don't see us reading anywhere. Is this the reason for removing it? And slightly reducing the message size?

Reply:
yes correct, today it has no use, so I just removed it. This header also is not being sent out so will save memory locally not in the messages sent

Reply:
👍